package.json

```diff
@@ -51,14 +52,14 @@
     "mini-css-extract-plugin": "1.6.1",
     "mini-svg-data-uri": "1.1.3",
     "nanoid": "3.3.3",
-    "node-sass": "7.0.1",
+    "node-sass": "7.0.3",
     "normalize.css": "8.0.1",
     "postcss-custom-properties": "11.0.0",
     "postcss-font-smoothing": "0.1.0",
     "postcss-loader": "6.1.1",
     "query-string": "3.0.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "react": "^18.2.0",
```

Comment:
At least in the `static/js/ktl-component/ktl-helpers.js` file, we use `hydrate`, that leads us to a warning.
<img width="1555" alt="Screenshot 2023-08-24 at 11 58 02" src="https://github.com/JetBrains/kotlin-web-site/assets/3338311/6f3d9cec-6b6c-462f-89ee-ec319699c67d">
Please, replace it with hydrateRoot.